custom_components/miele/translations/de.json

```diff
@@ -417,10 +416,10 @@
                             "dust_box_missing": "Fehlender Staubbehälter",
                             "blocked_drive_wheels": "Antriebsrad blockiert",
                             "blocked_brushes": "Bürsten blockiert",
```

Comment:
According to the error message description in the Scout RX3 Home Vision manual: `Überlastung Bürsten`